docs/tutorials/dev_add_test_group.md

```diff
@@ -0,0 +1,2538 @@
+            The test group that this test case belongs to
+
+        resolution : float
+            The resolution of the test case in km
+
+        name : str
+            The name of the test case
+        """
+        self.resolution = resolution
+        if resolution >= 1.:
+            res_str = f'{resolution:g}km'
+        else:
+            res_str = f'{resolution * 1000.:g}m'
+        subdir = os.path.join(res_str, name)
+        super().__init__(test_group=test_group, name=name,
+                         subdir=subdir)
+
+        self.add_step(
+            InitialState(test_case=self, resolution=resolution))
+
+    def validate(self):
+        """
+        Compare ``temperature``, ``salinity`` and ``layerThickness`` from the
+        initial condition with a baseline if one was provided
+        """
+        super().validate()
+        variables = ['temperature', 'salinity', 'layerThickness']
+        compare_variables(test_case=self, variables=variables,
+                          filename1='initial_state/initial_state.nc')
+
+```
+
+Now, we'll make `Default` descend from `YetAnotherChannelTestCase` and remove
+the redundant pieces.  Here's what's left:
+
+```python
+from polaris.ocean.tests.yet_another_channel import YetAnotherChannelTestCase
+from polaris.ocean.tests.yet_another_channel.forward import Forward
+from polaris.ocean.tests.yet_another_channel.viz import Viz
+from polaris.validate import compare_variables
+
+
+class Default(YetAnotherChannelTestCase):
+    """
+    The default test case for the "yet another channel" test group simply creates
+    the mesh and initial condition, then performs a short forward run on 4
+    cores.
+    """
+
+    def __init__(self, test_group, resolution):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : polaris.ocean.tests.yet_another_channel.YetAnotherChannel
+            The test group that this test case belongs to
+
+        resolution : float
+            The resolution of the test case in km
+        """
+        super().__init__(test_group=test_group, resolution=resolution,
+                         name='default')
+
+        self.add_step(
+            Forward(test_case=self, ntasks=4, min_tasks=4, openmp_threads=1,
+                    resolution=resolution))
+
+        self.add_step(
+            Viz(test_case=self))
+
+    def validate(self):
+        """
+        Compare ``temperature``, ``salinity``, ``layerThickness`` and
+        ``normalVelocity`` in the ``forward`` step with a baseline if one was
+        provided.
+        """
+        super().validate()
+        variables = ['temperature', 'salinity', 'layerThickness',
+                     'normalVelocity']
+        compare_variables(test_case=self, variables=variables,
+                          filename1='forward/output.nc')
+
+```
+
+### Adding a parameter study
+
+The `yet_another_channel` test group contains several test cases in addition
```

Comment:
```suggestion
The `baroclinic_channel` test group contains several test cases in addition
```

I think we want to keep this `baroclinic_channel` because the tutorial subject will not have added all these test cases by this point.